<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SqlSaveSettingForm</class>
 <widget class="QWidget" name="SqlSaveSettingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SqlSaveSettingForm</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>422</width>
        <height>297</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_addgroup"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>解决方案:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_pro"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_newpro"/>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_addnewpro">
       <property name="text">
        <string>新建工程</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_savedatasql">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>存档</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
